### ISSUE TITLE
Consistent camelCase in translations

### ISSUE THREAD

**Issue by tijlleenders** [tijlleenders/ZinZen#1499]
**Is your feature request related to a problem? Please describe.**
It's very annoying when naming conventions aren't present/respected.

**Describe the solution you'd like**
Have all translation items respect camelCase.

**Describe alternatives you've considered**
Leave it the way it is. This is going to cost more time/annoyance in the long run than this simple fix.

**Additional context**
None.

**Comment by vinaybadgujar102** [comment]
@tijlleenders 
For some translations, there are emojis in keys. Should the emojis be removed?
![image](https://github.com/tijlleenders/ZinZen/assets/90821253/1c50f699-9bfa-4550-ba0b-a545972d74c1)

**Comment by tijlleenders** [comment]
Please keep the emoji's.  
I don't see why they need to be removed.

**Comment by vinaybadgujar102** [comment]
I meant that even if we removed emojis from the key side of the json the value displayed on the UI would be same. Right @Tushar-4781?

**Comment by tijlleenders** [comment]
Yeah, sorry I should explain my motivation.   There are only a few keys that contain emoji's. They are for the 'default goals' you get on first start of the app.  
  
So yeah if we need to use/type these keys a lot I agree it makes sense to keep them in alpha-characters... but we don't.  
So I don't see any reason to take them out. Technically they are identical to Hindi characters right, both are unicode text. Copy/paste them once and it'll work.

**Comment by vinaybadgujar102** [comment]
Yeah, I agree with you. As it is only used once, we can keep it as it is.
